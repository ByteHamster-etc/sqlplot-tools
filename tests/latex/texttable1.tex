line1
% TEXTTABLE SELECT 5 + 2 AS f1, 6 * 9 AS x6x9, 'blah' AS f3
line2
% SQL SELECT 1
this is the end