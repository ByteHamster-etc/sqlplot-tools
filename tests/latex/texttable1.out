line1
% TEXTTABLE SELECT 5 + 2 AS f1, 6 * 9 AS x6x9, 'blah' AS f3
+----+------+------+
| f1 | x6x9 |   f3 |
+----+------+------+
|  7 |   54 | blah |
+----+------+------+
% END TEXTTABLE SELECT 5 + 2 AS f1, 6 * 9 AS x6x9, 'blah' AS f3
line2
% SQL SELECT 1
this is the end
